packages/x-components/src/x-modules/history-queries/components/my-history.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <BaseSuggestions
+    :suggestions="historyQueries"
+    class="x-my-history-queries"
+    data-test="my-history-queries"
+    :animation="animation"
+  >
+    <template #default="{ suggestion, index }">
+      <!--
+        @slot History Query item
+            @binding {Suggestion} suggestion - History Query suggestion data
+            @binding {number} index - History Query suggestion index
+      -->
+      <slot name="suggestion" v-bind="{ suggestion, index }">
+        <HistoryQuery
+          :suggestion="suggestion"
+          data-test="history-query-item"
+          class="x-history-queries__item"
+        >
+          <template #default="{ queryHTML }">
+            <!-- eslint-disable max-len -->
+            <!--
+              @slot History Query content
+                  @binding {Suggestion} suggestion - History Query suggestion data
+                  @binding {string} queryHTML - Suggestion's query with the matching part inside a span tag
+                  @binding {number} index - History Query suggestion index
+            -->
+            <!-- eslint-enable max-len -->
+            <slot name="suggestion-content" v-bind="{ suggestion, index, queryHTML }" />
+          </template>
+          <template #remove-button-content>
+            <!--
+              @slot History Query remove button content
+                  @binding {Suggestion} suggestion - History Query suggestion data
+                  @binding {number} index - History Query suggestion index
+            -->
+            <slot name="suggestion-remove-content" v-bind="{ suggestion, index }" />
+          </template>
+        </HistoryQuery>
+      </slot>
+    </template>
+  </BaseSuggestions>
+</template>
+
+<script lang="ts">
+  import { HistoryQuery } from '@empathyco/x-types';
+  import Vue from 'vue';
+  import { Component, Prop } from 'vue-property-decorator';
+  import { State, xComponentMixin } from '../../../components';
```

Comment:
I know I added this import, but it is wrong, barrel imports should be avoided in our code.